test/AutoDiff/autodiff_indirect_diagnostics.swift

```diff
@@ -107,4 +103,3 @@ extension TF_691: Differentiable where Scalar: Differentiable {}
 func identity<T>(_ x: TF_691<T>) -> TF_691<T> { x }
```

Comment:
Does a test for [TF-697](https://bugs.swift.org/projects/TF/issues/TF-697?filter=allopenissues) exist somewhere?

Reply:
Yes, it was added in https://github.com/apple/swift/pull/26420.